src/Shared/Data.php

```diff
@@ -695,14 +715,12 @@ public function getAllAvailablePaymentMethods($use_cache = true)
             } else {
                 delete_transient($transient_id);
             }
-
             // No cache exists, call the API and cache the result
             if ($methods === false) {
                 if (!$apiKey) {
                     return [];
                 }
-                $methods = $this->api_helper->getApiClient($apiKey)->methods->allAvailable($filters_key);
-
+                $methods = $this->api_helper->getApiClient($apiKey)->methods->allAvailable($filters);
```

Comment:
`$transient_id` is generated from `$filters_key`. Please double-check if it is okay to use different filters for a request.

Reply:
yes, I need different filters from before, cause now this will ask for the methods that we can show in admin and the ones that we can build, and I need the issuers. But all the others are not needed indeed.